newIDE/app/src/Export/Builds/BuildCard.js

```diff
@@ -64,46 +116,266 @@ export const BuildCard = ({
+                    )}
+                    <Column expand noMargin justifyContent="center">
+                      <Line noMargin justifyContent="end">
+                        {isOnlineBuild ? (
+                          <Text size="body2">
+                            <Trans>Current build online</Trans>
+                          </Text>
+                        ) : (
+                          game.acceptsBuildComments &&
+                          build.type === 'web-build' && (
+                            <LineStackLayout alignItems="center" noMargin>
+                              <div
+                                style={{
+                                  ...styles.openForFeedbackChip,
+                                  backgroundColor: gdevelopTheme.message.valid,
+                                }}
+                              />
+                              <Text size="body2">
+                                <Trans>Build open for feedbacks</Trans>
+                              </Text>
+                            </LineStackLayout>
+                          )
+                        )}
+                      </Line>
+                    </Column>
+                  </Line>
+                }
+              >
+                <Column expand noMargin>
+                  {windowWidth === 'small' && (
+                    <BuildAndCreatedAt build={build} />
+                  )}
+                  <Spacer />
+                  <Line noMargin>
+                    {isEditingName ? (
+                      <Line noMargin expand>
+                        <TextField
+                          ref={nameInput}
+                          style={styles.textField}
+                          value={name}
+                          margin="none"
+                          onChange={(_, value) => setName(value)}
+                          onBlur={() => {
+                            onBlurEditName(i18n);
+                          }}
+                          hintText={buildName}
+                          disabled={gameUpdating}
+                          onKeyPress={event => {
+                            if (shouldValidate(event) && nameInput.current)
+                              nameInput.current.blur();
+                          }}
+                          maxLength={50}
+                        />
+                        {gameUpdating && (
+                          <>
+                            <Spacer />
+                            <CircularProgress style={styles.circularProgress} />
+                          </>
+                        )}
+                      </Line>
+                    ) : (
+                      <Line noMargin alignItems="baseline">
+                        <Text noMargin>{buildName}</Text>
+                      </Line>
+                    )}
+                  </Line>
+                  <Line noMargin alignItems="center">
+                    <BackgroundText style={{ textAlign: 'left' }}>
```

Comment:
Nit: for styles that don't change, put them in a `styles` object and use `styles.xxx` to avoid creating a new object in memory at each render (putting pressure on the JS GC)